docs/data-sources/package_repository.md

```diff
@@ -0,0 +1,116 @@
+---
+Title: "Package Repository Resource"
+Description: |-
+    Creating the package Repository resource.
+---
+
+# Package Repository
+
+This resource allows you to read package repository from a cluster through Tanzu Mission Control.
+
+It's a Kubernetes resource which references Package Repository Bundle.It has information such as image url of Package Repository Bundle and necessary credentials to pull Package Repository Bundle.
+
+[package-repository]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-E0168103-7A6F-4C07-8768-19D9B1EB4EFA.html
+
+
+## Cluster scoped Package Repository
+
+### Example Usage
+
+```terraform
+# Read Tanzu Mission Control package repository : fetch cluster package repository details
+resource "tanzu-mission-control_package_repository" "read_cluster_pkg_repository" {
```

Comment:
Shouldn't this be `data "tanzu-mission-control_package_repository" "read_cluster_pkg_repository" {` instead of resource

Reply:
updated